src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -40,7 +40,7 @@ public class CertificateGenerator
         // Give the cert a grace period in case there's a time skew between machines
         private readonly TimeSpan _gracePeriod = TimeSpan.FromHours(1);
 
-        private const string _authorityCanonicalName = "DO_NOT_TRUST_WcfBridgeRootCA";
+        private const string _authorityCanonicalName = Constants.CertificateIssuerName;
         private const string _signatureAlthorithm = "SHA1WithRSAEncryption";
```

Comment:
CertificateGenerator shouldn't have any dependencies to `Constants` or the Bridge. If you want to constant-ize this, we should make `AuthorityCanonicalName` settable and ensure that every instance of `CertificateGenerator` is initialized with `CertificateGenerator.AuthorityCanonicalName = Constants.CertificateAuthorityName `

Otherwise, CertificateGenerator.AuthorityCanonicalName is effectively a const that you can use everywhere after initialization.